cns/kubecontroller/nodenetworkconfig/reconciler.go

```diff
@@ -164,6 +164,9 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager, node *v1.Node) error {
 		WithEventFilter(predicate.Funcs{
 			// check that the generation is the same - status changes don't update generation.
 			UpdateFunc: func(ue event.UpdateEvent) bool {
+				if ue.ObjectOld == nil || ue.ObjectNew == nil {
```

Comment:
is this possible in an update event?

Reply:
I copied this from the [`GenerationChangedPredicate`](https://github.com/kubernetes-sigs/controller-runtime/blob/5aa6a71fc79d4058b26355eb58bdf2430fce423e/pkg/predicate/predicate.go#L161-L172) Update handler so I think the answer is yes